sdk/storage/azure-storage-file-datalake/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Release History
 
-## 12.17.0 (Unreleased)
+## 12.17.0 (2024-09-17)
 
 ### Features Added
+- Stable release of features from 12.23.0b1
```

Comment:
C+P error, should be 12.17.0b1
This feedback applies to all packages that aren't Blob (since that's the source of the C+P)